analysed-packages/jquery-mousewheel/version-3.1.13/README.md

```diff
@@ -0,0 +1,17 @@
+## Download Location
+
+https://github.com/jquery/jquery-mousewheel/archive/refs/tags/3.1.13.zip
+
+## Package URL (purl)
+
+pkg:github/jquery/jquery-mousewheel@3.1.13?download_url=https://github.com/jquery/jquery-mousewheel/archive/refs/tags/3.1.13.zip
```

Comment:
To my knowledge the purl for this package is:
pkg:github/jquery/jquery-mousewheel@3.1.13

Reply:
Done, thank you!